python/ml4ir/applications/ranking/tests/test_t_test_calculation.py

```diff
@@ -0,0 +1,119 @@
+import numpy as np
+from scipy import stats
+import unittest
+import warnings
+from ml4ir.applications.ranking.model.ranking_model import perform_click_rank_dist_paired_t_test, compute_stats_from_stream
+
+warnings.filterwarnings("ignore")
+
+
+class TestTtestCalculation(unittest.TestCase):
+    """
+    Testing the calculation of paired t-test statistic and its p-value.
+    """
+
+    def t_test_calculation(self, a_bucket, b_bucket, batch_size):
+        n = len(a_bucket)
+        expected_t_test_stat, expected_pvalue = stats.ttest_rel(a_bucket, b_bucket)
```

Comment:
@mohazahran Shouldn't you be doing `ttest_ind` here?

Reply:
Since the two models are tested on the same splits, then a paired t-test would be suitable and hence ttest_rel. What do you think?